src/App.tsx

```diff
@@ -1,223 +1,101 @@
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React, { useContext, useEffect, useRef } from 'react';
+import React, {
+  useContext,
+  useEffect,
+  useState,
+} from 'react';
+import { getTodos } from './api/todos';
 import { AuthContext } from './components/Auth/AuthContext';
+import { Footer } from './components/Auth/todos/Footer';
+import { Header } from './components/Auth/todos/Header';
+import { TodoList } from './components/Auth/todos/TodoList';
+import { Todo } from './types/Todo';
 
 export const App: React.FC = () => {
-  // eslint-disable-next-line @typescript-eslint/no-unused-vars
   const user = useContext(AuthContext);
-  const newTodoField = useRef<HTMLInputElement>(null);
 
-  useEffect(() => {
-    // focus the element with `ref={newTodoField}`
-    if (newTodoField.current) {
-      newTodoField.current.focus();
+  const [addError, setAddError] = useState(true);
+  const [deleteError, setDeleteError] = useState(false);
+  const [updateError, setUpdateError] = useState(false);
+
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [visibleTodos, setVisibleTodos] = useState<Todo[]>([]);
+  const [status, setStatus] = useState('All');
+
+  const hasError = (isVisible?: boolean) => {
+    if (isVisible === false) {
+      setAddError(false);
+      setDeleteError(false);
+      setUpdateError(false);
+    }
+
+    return addError || deleteError || updateError;
+  };
```

Comment:
Create a separate component for error and move this logic there.

Also I see no implementation points from screen. Is it?
<img width="631" alt="Screenshot 2022-10-13 at 10 26 42" src="https://user-images.githubusercontent.com/17098736/195530300-be42c02d-0c9b-49dd-acd3-4bf5ebc85aea.png">